client/src/components/new-experience/index.tsx

```diff
@@ -1,38 +1,45 @@
 import React, { useEffect } from "react";
 import { FlexContainer, Flex } from "../../box/flex-box";
-import * as _ from "./new-experience";
 import { BlockSubTitle, BlockTitle } from "../../text/text";
 import AOS from "aos";
 import "aos/dist/aos.css";
+import {
+  SectionNewExperience,
+  NewExperienceImage,
+  NewExperienceImageTwo,
+  NewExperienceDescription,
+  NewExperienceIcon,
+  NewExperienceText,
+} from "./new-experience";
 
 const NewExperience = () => {
   useEffect(() => {
     AOS.init({ duration: 2000 });
   }, []);
 
   return (
-    <_.SectionNewExperience>
+    <SectionNewExperience>
       <FlexContainer data-aos="fade-down">
         <Flex>
-          <_.NewExperienceImage />
-          <_.NewExperienceImageTwo />
+          <NewExperienceImage />
+          <NewExperienceImageTwo />
```

Comment:
Well, could we combine `NewExperinceImage` and `NewExperienceImageTwo` in one component and change logic passing appropriate prop which would switching between Component representation?